app/components/project-card-skills.js

```diff
@@ -1,21 +1,73 @@
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+const {
+  Component,
+  set,
+} = Ember;
+
+/**
+  `project-card-skills` composes the given project's list of skills on the
+  project's card.
+
+  ## default usage
+
+  ```Handlebars
+  {{project-card-skills skills=project.skills}}
+  ```
+
+  @class project-card-skills
+  @module Component
+  @extends Ember.Component
+ */
+export default Component.extend({
   classNames: ['project-card-skills'],
+
+  /**
+    Returns whether or not the overflow skills are hidden.
```

Comment:
I'm confused as well. It seems that if you call `skillItemHidden()` in the handlebars for `skill-list-items` it will trigger `skillItemHidden()` in this component, which just sets it to true.

So right now there isn't any criteria, it's just set and there isn't a value set to regulate how many skills are visible, so quite frankly I'm unsure how this control even works right now. Seems to me it will just display a bunch of skills regardless of how many there are and offer a toggle that is useless.

[project-card-members ](https://github.com/code-corps/code-corps-ember/blob/develop/app/components/project-card-members.js) handles this a lot better, though its does it through a constant which doesn't give is much control. Ideally, I feel it should be a variable that we can set via handlebars or not set and have it use a default value.